### DISCUSSION TITLE
Have upstream_branch default to the repository's default branch

### DISCUSSION THREAD

**Discussion by ebronson68** [AndreasAugustin/actions-template-sync#341] [Polls]
This is a copy of an existing issue here: https://github.com/AndreasAugustin/actions-template-sync/issues/338

If everyone could please vote on this poll to get it implemented to this repository, it would be greatly appreciated!

Thanks!

# Description

Default to the default branch in the repository rather than require the user to override the default, "main", with the name of the branch they normally base their PRs from.

**Comment by AndreasAugustin** [comment]
@ebronson68 thanks a lot for the poll.
Let's give the community the chance to vote and give comments till 19th May. What do you think?

**Reply by ebronson68** [reply]
Sounds good to me!

**Comment by AndreasAugustin** [comment]
Reopened #338 because of positive feedback

**Reply by AndreasAugustin** [reply]
released with version [v1.0.0-draft](https://github.com/AndreasAugustin/actions-template-sync/releases/tag/v1.0.0-draft)
